packages/backend/src/tools/Logger.ts

```diff
@@ -33,9 +33,9 @@ export class Logger {
     return this.configure({ service: object.constructor.name })
   }
 
-  error(error: unknown) {
+  error(error: unknown, annotation?: string) {
     if (this.options.logLevel >= LogLevel.ERROR) {
-      const message = getErrorMessage(error)
+      const message = [annotation, getErrorMessage(error)].join(' ')
```

Comment:
This will produce messages that have an extra `" "` at the start

Reply:
Nice catch.